<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_heading">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Graphical user interface!!!!!!</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>258</width>
      <height>810</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QDial" name="dial_2"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;kjldskhjlkjljkh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="dial_4"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="txt_browser_one"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item>
      <widget class="QDial" name="dial"/>
     </item>
     <item>
      <widget class="QListWidget" name="lst_widget"/>
     </item>
     <item>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_text_browser_2">
       <property name="text">
        <string>Combo box</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_one"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Progress bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_progress_complete">
       <property name="text">
        <string>Progress Complete!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progbar_one">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>127</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_list_view">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>List View </string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>254</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>254</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>DESTROY</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_text_browser">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text Browser</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_3">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>263</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuE_dit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuH_elp">
    <property name="title">
     <string>H&amp;elp</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuE_dit"/>
   <addaction name="menuH_elp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
